dashboard/routings/result.py

```diff
@@ -56,6 +57,11 @@ def result(end, index=0):
         page = 'endpoint/endpoint_plotly.html'
         graph = get_time_per_user(end)
 
+    # returns a page with a table containing the outliers.
+    elif index == 7:
```

Comment:
i'm not sure of these magic numbers in the code. 
can't we define symbolic constants for them?